examples/07_joining_tables_with_FuzzyJoin.py

```diff
@@ -0,0 +1,161 @@
+"""
+Joining tables with fuzzyjoin
+================================
+
+In this example, we show how to join tables with the :function:`fuzzyjoin` function. 
+We also demonstrate why this method is the most easy and appropriate tool for handling 
+the joining of tables for users that want to improve their machine learning models quickly.
+We go through some of the rich options that allow for manipulation when joining tables,
+such as taking into account the precision of the match.
+
+To do so, we will predict the happiness score of a country from
+the 2022 [World Happiness Report](https://worldhappiness.report/).
+We will also use data provided from the [World Bank open data platform](https://data.worldbank.org/)
+in order to create a satisfying prediction model.
+
+"""
+
+###############################################################################
+# Data Importing and preprocessing
+# --------------------------------
+#
+# We import the happiness score data first:
+import pandas as pd
+
+df = pd.read_csv('https://raw.githubusercontent.com/jovan-stojanovic/data/main/Happiness_report_2022.csv', thousands=',')
+df.drop(df.tail(1).index, inplace = True)
+
+# Let's take a look at the table:
+df.head(3)
+ 
+#################################################################
+# The Happiness score was computed using the Gallup World Poll survey results. 
+# The report stress out some of the possible explanatory factors: GDP per capita, Social support, Generosity etc.
+# However, these factors here are only estimated indexes used to calculate the happiness score.
+
+# The sum of all explanatory indexes is then the happiness score itself:
+df['Sum_of_factors'] = df.iloc[:,[5,6,7,8,9,10,11]].sum(axis=1)
+df[['Happiness score','Sum_of_factors']].head(3)
+
+# Thus, we cannot use them for our prediction model.
+X = df[['Country']]
+y = df[['Happiness score']]
+
+# If we want to create a machine learning model which predicts the happiness index of any new country or future date,
+# we will need to include explanatory factors from other tables.
+
+###############################################################################
+# Joining tables using fuzzyjoin
```

Comment:
It's strange to have a section title with the same name as the page title. Let's change (improve) one of the two